app/src/main/java/com/edulectronics/tinycircuit/Models/Factories/ComponentFactory.java

```diff
@@ -19,7 +19,7 @@ public static Component CreateComponent(String name) {
             case "Gloeilamp":
                 return new Lightbulb();
             case "Weerstand":
```

Comment:
@bvdhoek kan ook, nu heb ik het gemaakt dat er maar 1 value is, zoals je in de nieuwste commit kan zien